packages/flutter/test/rendering/recording_canvas.dart

```diff
@@ -160,8 +160,8 @@ class _MethodCall implements Invocation {
 }
 
 String _valueName(Object value) {
-  if (value is double)
-    return value.toStringAsFixed(1);
+  if (value is Offset)
+    return 'Offset(${value.dx}, ${value.dy})';
```

Comment:
I think we should leave this as it was and fix https://github.com/flutter/flutter/issues/1221 properly instead.